redash/query_runner/pg.py

```diff
@@ -146,7 +146,7 @@ def _get_connection(self):
                                       port=self.configuration.get('port'),
                                       dbname=self.configuration.get('dbname'),
                                       sslmode=self.configuration.get('sslmode'),
-                                      async=True)
+                                      async_=True)
```

Comment:
__async__ is a reserved work in Python 3.7 and later so psycopg2 made a change to avoid the syntax error https://github.com/psycopg/psycopg2/releases/tag/2_7